workshops/management/commands/check_for_workshop_websites_updates.py

```diff
@@ -137,25 +137,25 @@ def load_from_github(self, github, repo_url, default_branch='gh-pages'):
         branch = repo.get_branch('gh-pages')
         return branch
 
-    def detect_changes(self, branch, event, save_tags=False):
-        """Detect changes made to event's meta tags."""
+    def detect_changes(self, branch, event, save_metadata=False):
+        """Detect changes made to event's metadata."""
         changes = []
 
         # compare commit hashes
         if branch.commit.sha != event.repository_last_commit_hash:
-            # Hashes differ? Update commit hash and compare stored tags
+            # Hashes differ? Update commit hash and compare stored metadata
             event.repository_last_commit_hash = branch.commit.sha
 
-            tags_new = self.get_event_tags(event.url)
+            new_metadata = self.get_event_metadata(event.url)
```

Comment:
I would probably use `metadata_new` and `metadata_old` to be consistent with other usage.